scripts/measure-change-events.sh

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env bash
+
+usage () {
+    echo "$(basename $0) [-h]"
+    echo ""
+    echo "Count lines of code, lines of specification, and other metrics for"
+    echo "the OpenSUT and the two change events."
+    echo ""
+    echo "Options:"
+    echo ""
+    echo "-h    Print this help message."
+    echo ""
+}
+
+while getopts "h" opt; do
+  case $opt in
+    h)
+      usage
+      exit 0
+      ;;
+    \?)
+      echo "Invalid option: -$OPTARG" >&2
+      exit 1
+      ;;
+  esac
+done
+
+shift $((OPTIND -1))
+
+# -----------------------------------------------------------------------------
+# Count specs and lines of code at a given point in git history.
+
+# Counts lines of source code, excluding whitespace and comments.
+#
+# params: target files/directories
+count_sloc () {
+    cloc --include-lang="C,C/C++ Header" $@ | \
```

Comment:
The script depends on `cloc` which does not seem to be installed by default. It is also not clear if I am supposed to run this on linux/mac or both - please add some instructions about how to install dependencies / what are the system requirements.